core/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/list/TaskInfo.java

```diff
@@ -50,17 +51,24 @@
 
     private final String description;
 
+    private final long startTime;
```

Comment:
Err, the next one a TimeValue, this one a DateTime or something? Its not required, at all and I just thought of it but a TimeValue would make runningTimeNanos easier to read I think.